Added Gmail Policy Group 19 (#324)
cisagov/ScubaGoggles: baselines/Gmail Minimum Viable Secure Configuration Baseline v0.2.md

    - [T1566:001: Phishing: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
  - [T1534: Internal Spearphishing](https://attack.mitre.org/techniques/T1534/)

#### GWS.GMAIL.19.2v0.1
Domains SHALL NOT be added to lists that bypass spam filters and hide warnings.

- _Rationale:_ Allowing an entire domain to bypass the spam filters and hide warnings allows for the potential for a spoofed email within the domain to bypass the filter and prevents the user from knowing. Not adding domains and users helps mitigate the risk.
- _Last modified:_ April 10, 2024

- MITRE ATT&CK TTP Mapping
  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
    - [T1566:001: Phishing: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
  - [T1534: Internal Spearphishing](https://attack.mitre.org/techniques/T1534/)

#### GWS.GMAIL.19.3v0.1
Bypass spam filters and hide warnings for all messages from internal and external senders SHALL NOT be enabled.

> - _Rationale:_ Bypassing spam filters and hiding warning for all messages from internal and external senders creates a security risk because of the potential for a malicious message being able to bypass filters. Disabling this feature mitigates the risk.

Reviewer: Recommend tweaking this rationale because this setting effectively disables spam filtering, creating more than the potential for messages to bypass filters.
```suggestion
- _Rationale:_ Bypassing spam filters and hiding warning for all messages from internal and external senders creates a security risk because all messages are allowed to bypass filters. Disabling this feature mitigates the risk.
```

- _Last modified:_ April 10, 2024

- MITRE ATT&CK TTP Mapping
  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
    - [T1566:001: Phishing: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
  - [T1534: Internal Spearphishing](https://attack.mitre.org/techniques/T1534/)

### Resources

> -   [How to bypass the spam filter for incoming emails using the spam settings ](https://knowledge.workspace.google.com/kb/how-to-bypass-the-spam-filter-for-incoming-emails-using-the-spam-settings-000006661)

Reviewer: This link gives me a 404 error

Author: Fixed the link

### Prerequisites

-   N/A

### Implementation

To configure the settings for spam filtering:

#### Policy Group 19 Common Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Apps -\> Google Workspace -\> Gmail**.
3.  Select **Spam, Phishing, and Malware**.

#### GWS.GMAIL.19.1v0.1 Instructions
1.  Un-select **Bypass spam filters for messages from senders or domains in selected lists.**
2.  Select **Save**.

#### GWS.GMAIL.19.2v0.1 Instructions

> 1.  Un-select **Bypass spam filters and hide warnings for messages from senders or domains in selected lists.**

Reviewer 2: Policy Group 19 Common Instructions
	1. Sign in to the [Google Admin Console](https://admin.google.com/).
	2. Select Apps -> Google Workspace -> Gmail.
	3. Select Spam, Phishing, and Malware.
	4. Under Spam Select Add Another Rule

GWS.GMAIL.19.1v0.2 Instructions
	1. Un-select Bypass spam filters for messages from senders or domains in selected lists.
	2. Or select Bypass spam filters for messages from senders or domains in selected lists and select a list that does not include entire Domains 
	3. Select Save.
GWS.GMAIL.19.2v0.2 Instructions
	1. Un-select Bypass spam filters and hide warnings for messages from senders or domains in selected lists.
	2. Or select Bypass spam filters and hide warnings for messages from senders or domains in selected lists and select a list that does not include entire Domains 
	4. Select Save.
GWS.GMAIL.19.3v0.2 Instructions
	1. Un-select Bypass spam filters and hide warnings for all messages from internal and external senders
	2. Select Save.
	
@adhilto Thoughts on these implementation steps? I believe that they are correct and satisfy the conditions of either unselect/disabled the setting or enabled/selected and configured without domains in the list
